requirements.txt

```diff
@@ -5,7 +5,6 @@ celery[pytest]==5.2.7
 click==8.1.3
 cryptography~=3.4.8
 dask==2022.8.0
-emails
```

Comment:
This is unused as far as I can tell so I'm removing it here as part of this housekeeping.